protos/dcs/mission/v0/mission.proto

```diff
@@ -303,6 +303,20 @@ message StreamEventsResponse {
     dcs.common.v0.Airbase place = 3;
   }
 
+  // Occurs when a chat message is sent on the server
+  message PlayerSentChatEvent {
```

Comment:
Would `PlayerSendChatEvent` (`Send` instead of `Sent`) maybe a bit more consistent naming compared to the other event names? Wdyt?

Reply:
My habit from other environments leaked here -- an "event" should as past tense; I didn't triple check the consistency on other events. 

If we are trying to convince other developers to move over to this dependency compared to the in-game stuff, yeah, keeping it close to the original DCS API scheme (active over past) would make sense.

I'll update that name -- and during an API review we want to change the convention, we uplift everything at the same time.

Reply:
I think https://github.com/DCS-gRPC/rust-server/pull/89/commits/de2be5b7ab35969973fbc4d10163478aa7848c6b is missing the following change:

```suggestion
  message PlayerSendChatEvent {
```